### PR TITLE
Added Bug Reporting guidelines.

### PR DESCRIPTION
**Pull request by Kiroto**
This helps when people report bugs, saving the team time and resources.

### CODE REVIEW

**Review by thecatcore — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -28,6 +28,21 @@ A: In the configs folder there is a `raa.json5` document. If you set the `"regen
...
+A: You have two options. If the bug seems like something minor and doesn't heavily impact gameplay, you may want to ask in the Discord Server first! (#raa-mod-talk or #raa-brainstorming).
+
+However, if the bug causes a crash or heavily impacts gameplay, you should post an issue that contains:
+
+- Operative System (Commonly Windows)
+- Minecraft Console Log
+
+The contents of these files:
+- config/raa.json5
+- config/raa/materials.json
```
Comment (oliviathevampire) [inline README.md]: should be config/raa/materials/materials_0.json

File: README.md [hunk 2]
```diff
@@ -28,6 +28,21 @@ A: In the configs folder there is a `raa.json5` document. If you set the `"regen
...
+- config/raa/dimensions.json
```
Comment (oliviathevampire) [inline README.md]: should be config/raa/dimensions/dimensions_0.json

File: README.md [hunk 3]
```diff
@@ -28,6 +28,21 @@ A: In the configs folder there is a `raa.json5` document. If you set the `"regen
...
+- config/raa/entities.json
```
Comment (oliviathevampire) [inline README.md]: should be config/raa/entities/entities_0.json

**Review by oliviathevampire — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (Kiroto) [reply README.md]: Adressed in 
68a5cb5

File: README.md (hunk 2 above)
Comment (Kiroto) [reply README.md]: Addressed in 
68a5cb5

File: README.md (hunk 1 above)
Comment (Kiroto) [reply README.md]: Addressed in 
68a5cb5

**Review by oliviathevampire — CHANGES_REQUESTED** [review]
materials_0 file should be in config/raa/materials
dimensions_0 should be in config/raa/dimensions
entities_0 should be in config/raa/entities

**Review by oliviathevampire — APPROVED** [review]

**Pull request merged**